### ISSUE TITLE
[Bug] - Dropdown text isn't cleared after clicking suggestion

### ISSUE THREAD

**Issue by tburrows13** [LederCards/spotlight#81]
### Prerequisites

- [X] I have searched the issue tracker to check if the issue has already been reported.

### Describe the bug

When searching for multiple tags, the text isn't cleared when a suggestion is clicked.

### Expected behavior

- Start typing "adv"
- Click on "Advocate" to add it to the list of selected tags
- "adv" text should be removed

### Current behavior

- "adv" text must be manually deleted with backspace before starting to type the next tag
<img width="1125" alt="Screenshot 2024-06-18 at 18 25 07" src="https://github.com/LederCards/spotlight/assets/22625968/78623a41-f195-44aa-8cab-401bd3856b2c">

### To Reproduce

See above

### Platform

Desktop

**Comment by tburrows13** [comment]
Oops, duplicate of #74
